hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/hss/HssCallAttempt.java

```diff
@@ -149,4 +155,47 @@ public boolean isScheduleRedirect() {
         }
         return null;
     }
+
+    /**
+     * Extracts the key set for scheduled calls.
+     *
+     * @return the key set
+     */
+    public Set<Key> keySetFor() {
```

Comment:
I see  why you moved this here but I do not think you really need this.  This is for getting the `authorizingKeys` which I do not think are used when creating schedule transactions.

Reply:
It is actually used to build the schedule signatories.